src/components/Comments/GetComment/SingleComment.tsx

```diff
@@ -0,0 +1,82 @@
+import React from "react";
+import { Box, Button, makeStyles, Typography } from "@material-ui/core";
+import TabViewIntroSection from "../../TabViewIntroSection";
+
+const getSingleCommentEndpoint = ["GET /comments/commentId"];
+const getSingleCommentHeading = "GET single comments";
+const getSingleCommentSubtitle = "Need to get just one comment?";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 1,
+    backgroundColor: theme.palette.background.paper,
+  },
+  buttonGroup: {
+    maxWidth: 800,
+  },
+  button: {
+    marginLeft: 10,
+    marginRight: 10,
+  },
+}));
+
+const SingleComment = () => {
+  const classes = useStyles();
+
+  const handleFlagFilter = (event: any) => {};
+
+  const handleRefIdFilter = (event: any) => {};
+
+  const handleOwnerIdFilter = (event: any) => {};
+
+  const handleOriginFilter = (event: any) => {};
+
+  return (
+    <Box mt={10} mb={5}>
+      <TabViewIntroSection
+        endpoints={getSingleCommentEndpoint}
+        heading={getSingleCommentHeading}
+        subtitle={getSingleCommentSubtitle}
+      ></TabViewIntroSection>
+      <Box mt={5}>
```

Comment:
Same thing here, absolute values are risky and inconsistent because screen devices aren't the same sizes.